src/Options/docs.js

```diff
@@ -20,7 +20,7 @@
  * @property {Adapter<AnalyticsAdapter>} analyticsAdapter Adapter module for the analytics
  * @property {String} appId Your Parse Application ID
  * @property {String} appName Sets the app name
- * @property {AuthAdapter[]} auth Configuration for your authentication providers, as stringified JSON. See http://docs.parseplatform.org/parse-server/guide/#oauth-and-3rd-party-authentication
+ * @property {Auth} auth Configuration for your authentication providers, as stringified JSON. See http://docs.parseplatform.org/parse-server/guide/#oauth-and-3rd-party-authentication
```

Comment:
Could you give some insight about this change? It looks like a breaking change, or was the type incorrect?

Reply:
I think it's an incorrect type, the Auth option is not an array, it's an object with `keys`:

```js
auth: {
  facebook: {},
  myAuth: {},
}
```

Reply:
Could you check:
- What do the docs (API docs and in `docs` repo) currently say - array or object?
- Are there any tests that currently use an array, or do they all use objects?
- If a developer currently set an array instead of object, would Parse Server accept that option and would that actually work?